test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+      // only one object
+      assertIsNull((Object) null);
+      assertNotNull(1);
+      assertNotNull("abc");
+
+      // some multiple objects
+      assertNotNull("abc", 1, "ABC");
+      assertIsNull("abc", 1, null);
```

Comment:
Try to group true cases and false cases separately.
eg. // Multilple objects with no null
and
// Multiple objects with null as the last element